torch/csrc/jit/import.cpp

```diff
@@ -228,6 +243,22 @@ void ScriptModuleDeserializer::convertModule(
       module->register_parameter(param_def.name(), tensor, /*is_buffer=*/false);
     }
   }
+  for (int i = 0; i < module_def.attributes_size(); ++i) {
+    const torch::AttributeDef& attr_def =
+        module_def.attributes(module_def.attributes_size() - 1 - i);
```

Comment:
Iterating backward here is subtle. Using attribute_id's will resolve this though.